templates/eks-target-roles-stackset.yml

```diff
@@ -0,0 +1,115 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+
+Parameters:
+  EKSExecutionRoleName:
+    Type: String
+    Default: "crowdstrike-eks-execution-role"
+  EventBridgeRoleName:
+    Type: String
+    Default: "crowdstrike-eks-eventbridge-role"
+  EventBusAccount:
+    Type: String
+  EventBusName:
+    Type: String
+    Default: "crowdstrike-eks-eventbus"
+  CodeBuildRoleName:
+    Type: String
+    Default: "crowdstrike-eks-codebuild-role"
+  PermissionsBoundary:
+    Type: String
+    Description: The name of the policy used to set the permissions boundary for IAM roles.
+    Default: ''
+
+Conditions:
+  SetPermissionsBoundary: !Not [ !Equals [ !Ref PermissionsBoundary, '' ] ]
+
+Resources:
+  EventBridgeRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      RoleName: !Ref EventBridgeRoleName
```

Comment:
Avoid hardcoding the name, unless absolutely needed. If it is needed, add exemption with explanation why this is needed?